src/ixdat/techniques/ms.py

```diff
@@ -543,7 +543,9 @@ def calc_n_dot_0(self, gas=None, w_cap=None, h_cap=None, l_cap=None, T=None, p=N
             p = self.p
 
         pi = np.pi
-        eta = DYNAMIC_VISCOSITIES[gas]  # dynamic viscosity in [Pa*s]
+        _eta_v = list(DYNAMIC_VISCOSITIES[gas].values())  # list of eta(T) for 'gas'
+        _eta_T = list(DYNAMIC_VISCOSITIES[gas].keys())  # list of Ts for 'gas'
+        eta = np.interp(T, _eta_T, _eta_v)  # dynamic viscosity of gas at T in [Pa*s]
```

Comment:
So I wondered what would happen if the T was out of range of the interpolation input, turns out `np.iterp` coerces to the value at the extremes, `_eta_v[0]` or `_eta_v[-1]`. In order to let the user know that they are going into unchartered waters, I think this is the right sort of thing for a Python warning. It might look something like this (using the already sorted list of T's:
```python
import warnings
...
# On the already sorted list of T
if T < _eta_T[0] or T > _eta_T[-1]:
    warnings.warn(f"Insufficient data in constants.py to appropriately estimate the dynamic viscosity for {gas} at temperature: {T}K")
```

Reply:
Yes exactly, and I havent used warnings before so I thought of using a try except but that seemed wrong and I decided not to. 

I will try and implement a warning.

Reply:
Look good